tests/behat/features/web/general/mediapackage.feature

```diff
@@ -19,7 +19,7 @@ Feature:
 
     And there are media package files:
       | id | name       | category     | extension | active | file   | flavor     | author        |
-      | 1  | Dog (😊🐶) | Animals      | png       | 1      | 1.png  | pocketcode | Bob Schmidt   |
+      | 1  | Dog (😊🐶)   | Animals      | png       | 1      | 1.png  | pocketcode | Bob Schmidt   |
```

Comment:
using php storm you can reformat feature files, then everything should be aligned in a nice way

Reply:
They are formatted the right way in phpstorm, the problem here are the emojis which have a different width across the different editors, that's why it looks like it's unformatted here on Github.